chemprop/uncertainty/predictor.py

```diff
@@ -46,11 +48,46 @@ def _calc_prediction_uncertainty(
 
 @UncertaintyPredictorRegistry.register("ensemble")
 class EnsemblePredictor(UncertaintyPredictor):
-    def _calc_prediction_uncertainty(
-        self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer
-    ) -> Tensor:
-        ...
-        return
+    """
+    Class that predicts uncertainty for predictions based on the variance in predictions among
+    an ensemble's submodels.
+    """
+    def __init__(self, individual_ensemble_predictions):
+        self.individual_ensemble_predictions = individual_ensemble_predictions
```

Comment:
This should be calculated in `_calc_prediction_uncertainty` rather than passing this into the `EnsemblePredictor` object.

Reply:
From my understanding, that variable is a boolean that determines whether the predictions from each model in the ensemble are returned or just the mean. Maybe a different name should be used for that variable?

Reply:
I see. How about always returning the individual results and letting people calculate the mean themselves? That would also relate to how we design the CLI, and I think this way makes sense as that is what we currently do in the prediction CLI (https://github.com/chemprop/chemprop/blob/main/chemprop/cli/predict.py#L284).

Reply:
@joelnkn If this makes sense to you, you could remove the `__init__()` and modify the output accordingly.